### ISSUE TITLE
Need ttnn support for Normalization op

### ISSUE THREAD

**Issue by Sudharsan-V** [tenstorrent/tt-metal#19772]
The pipeline for the SwinV2-S model includes a normalize operation. The current TTNN op library supports normalizing either across the entire tensor or on the last two dimensions, specifically for 4D tensors. However, the Swin model requires normalize to be applied only along the last dimension of a 4D tensor.

Torch link: [normalize](https://pytorch.org/docs/stable/generated/torch.nn.functional.normalize.html)

**Comment by Sudharsan-V** [comment]
cc: @saichandax , @mbahnasTT

**Comment by punithsekar** [comment]
Even in the yolo8s-world model, We need this support.
To normalize either dimension = 1 or dimension = -1.

**Comment by mbahnasTT** [comment]
@punithsekar what's the dimension of the tensor?
How about using LayerNorm or RMS or GroupNorm?

**Comment by punithsekar** [comment]
@mbahnasTT , 
The dimensions are:

I want to normalize two tensors
x = F.normalize(x, dim=1, p=2) # X_shape=[1,512,80,80]
w = F.normalize(w, dim=-1, p=2) ## w_shape=[1,80,512]

@Sudharsan-V , Thanks for sharing your thoughts. Even I feel the same. The normalization that I need to do is for a specific dimension. So, I don't find any workaround.

**Comment by ssinghalTT** [comment]
@Sudharsan-V @punithsekar can you try the attached file. 

[normalize.txt](https://github.com/user-attachments/files/20606812/normalize.txt)

sorry, could not upload as .py

**Comment by punithsekar** [comment]
Hi @ssinghalTT ,
The workaround provided works fine with yolov8s pipeline.
Thank you!

I have made some changes to the function w.r.t creating ones tensor (instead of creating a torch ones and converting to ttnn tensor, i have created ones tensor as ttnn tensor),

https://github.com/tenstorrent/tt-metal/compare/main...punith/yolov8s_world_update#diff-854365926009d3fd323aee955601879109abd94d14c39038c5dc5d8dd0d502ed

**Comment by Sudharsan-V** [comment]
Hi @ssinghalTT, the workaround provided is working fine for the SwinV2-S model.
Thank you!!

Closing this ticket as the workaround is working for both models.
